MapboxMobileEvents/Categories/NSUserDefaults+MMEConfiguration.m

```diff
@@ -285,12 +285,16 @@ - (NSString *)mme_userAgentString {
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
 
-        userAgent = [NSString stringWithFormat:@"%@/%@ (%@; v%@)",
+
```

Comment:
Given this is directly grabbing main bundle and UIDevice, testability here is difficult.

Options for boosting testability:
1. Include an additional function which takes Bundle/UIDevice arguments (Potentially another class/interface) to allow for mocking known inputs and potential problematic info for verification.

Reply:
it uses mme_mainBundle so you can inject that in tests at least

Reply:
We have `NSProcessInfo+SystemInfo` to use for this